src/main/java/github/scarsz/discordsrv/listeners/DiscordChatListener.java

```diff
@@ -338,15 +340,23 @@ private boolean processConsoleCommand(GuildMessageReceivedEvent event, String me
         }
 
         // log command to console log file, if this fails the command is not executed for safety reasons unless this is turned off
+        DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+        LocalDateTime now = LocalDateTime.now();
+        File dataFolder = DiscordSRV.getPlugin().getDataFolder();
+        if (!dataFolder.exists()) dataFolder.mkdir();
+        File logFolder = new File(dataFolder.getAbsolutePath() + File.separator + "logs");
+        if (!logFolder.exists()) logFolder.mkdir();
+        String fileName = logFolder + File.separator + "console-log-" + dtf.format(now) + ".log";
```

Comment:
Same changes here as the other file.
Also copy pasting code like this usually isn't a good idea; central method would be nice eg. `File getLogFile()`